src/components/IndividualPageContent/ContentRow.tsx

```diff
@@ -9,22 +9,27 @@ type ContentRowProps = {
   value: React.ReactNode;
   tooltip?: React.ReactNode;
   i?: any;
+  container?: boolean;
+  sx?: SxProps<Theme>;
 };
 
 export default function ContentRow({
   title,
   value,
   tooltip,
   i,
+  container,
+  sx,
```

Comment:
should this be optional prop?

Reply:
they are marked as optional in `ContentRowProps`

Reply:
my bad, missed it!